cdap-security/src/main/java/co/cask/cdap/security/server/LDAPAuthenticationHandler.java

```diff
@@ -35,19 +33,14 @@
   private static final List<String> mandatoryConfigurables = ImmutableList.of("debug", "hostname", "port", "userBaseDn",
                                                                               "userRdnAttribute", "userObjectClass");
 
-  private static final List<String> optionalConfigurables = ImmutableList.of("bindDn", "bindPassword", "useLdaps",
+  private static final List<String> optionalConfigurables = ImmutableList.of("bindDn", "useLdaps",
                                                                              "userIdAttribute", "userPasswordAttribute",
                                                                              "roleBaseDn", "roleNameAttribute",
                                                                              "roleMemberAttribute", "roleObjectClass");
-  private static boolean ldapSSLVerifyCertificate = true;
 
-  /**
-   * Create a new Authentication handler to use LDAP for external authentication.
-   */
-  @Inject
-  public LDAPAuthenticationHandler(CConfiguration configuration) throws Exception {
-    super(configuration);
-  }
+  private static final List<String> optionalSecureConfigurables = ImmutableList.of("bindPassword");
```

Comment:
Pushed a last commit which handles that: https://github.com/caskdata/cdap/pull/7213/commits/fda814ec77d3ed44921c2baa18763000f80d242c.